app/assets/javascripts/admin/models/flagged-post.js.es6

```diff
@@ -136,7 +136,7 @@ export default Post.extend({
             label: I18n.t("yes_value"),
             class: "btn-danger",
             callback() {
-              Post.deleteMany(replies.map(r => r.id))
+              Post.deleteMany(replies.map(r => r.id), true)
```

Comment:
I believe I mentioned this on the previous PR, but dislike APIs that take boolean parameters like this. If you read this line, it's totally unclear what `true` means. 

Please replace this with an object like `{ deferFlags: true }`

Reply:
I did remember that from last PR. I was looking for named parameters in JavaScript similar to what we have in Ruby and landed on [this post](http://2ality.com/2011/11/keyword-parameters.html).

> JavaScript does not have native support for named parameters like Python and many other languages.

I was not sure about the ideal using object literal in parameters but now that you have confirmed it, I'll go ahead and update the code to use it.

Reply:
Thanks @techAPJ